packages/client/src/__tests__/AuthenticatedClient.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { setupAuthenticatedClient } from '../__helpers__/setupAuthenticatedClient';
+import {
+  setupTestWallet,
+  testWalletProfileId,
+} from '../authentication/__helpers__/setupTestWallet';
+import { ProxyAction } from '../proxy-action';
+import { Reactions, ReactionTypes } from '../reactions';
+import { Transaction } from '../transaction';
+
+describe(`Given the authenticated LensClient configured to work with sandbox`, () => {
+  const wallet = setupTestWallet();
+  const getClient = setupAuthenticatedClient(wallet);
```

Comment:
all tests that require authentication in a single file, this way there is one user session for all tests, it's not ideal but it works. any better ideas?